src/markdown/ndoch_2016.md

```diff
@@ -2,6 +2,10 @@
 
 > Welcome to Code for Denver's 2016 National Day of Civic Hacking. This page aims to provide you with resources to help you have a successful and enjoyable event.
 
+## Location
+
+We will be at CA Technologies (formerly Rally Software) offices at **1550 Wynkoop St** in Lodo.
```

Comment:
Actually it is at the commons on Champa (I'm not sure if we'll need this info here though since we will already be there)